sinonjs/sinon
# Resolve #2305 by fixing broken 'matchers' link

#### Purpose (TL;DR) 
Resolve #2305 by fixing broken 'matchers' link

 #### Solution 
Fix broken links (3 of them)


 #### How to verify 
1. Check out this branch
2. Open file
3. Observe markdown
4. Hover over links to confirm accuracy

 #### Checklist for author
